Tetris.js

```diff
@@ -45,39 +45,37 @@ class Gameboard {
     startGame() {
         const currentSquarePos = { x: 4, y: 0 };
 
-        const s1 = new Square(
+        let s1 = new Square(
```

Comment:
Time to rename s1 to something like "currentActiveSqaure"?

Reply:
done